README.md

```diff
@@ -1,3 +1,224 @@
-# mod1
-mod1
-this is added
\ No newline at end of file
+<div>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
- [ ] Please consider Updating the Microverse logo with yours if possible or remove it totally. The template is designed by Microverse for us all to use to professionally document our projects and has it filled with content related to each project we may be working on.

Reply:
There is no logo in the read me file

Reply:
Sorry, @demesameneshoa, I must have to mismatched it with another project. Please address the other changes and request a review. You got this :hand: